code/game/ai_dmnet.c

```diff
@@ -2413,11 +2411,9 @@ int AINode_Battle_Retreat(bot_state_t *bs) {
 		AIEnter_Seek_LTG(bs, "battle retreat: enemy dead");
 		return qfalse;
 	}
-	//if there is another better enemy
-	if (BotFindEnemy(bs, bs->enemy)) {
-#ifdef DEBUG
+	//if there is another better enemy (only available in Developer mode)
+	if (BotFindEnemy(bs, bs->enemy) && (bot_developer.integer & DEVMODE_FOR_CODERS)) {
```

Comment:
BotFindEnemy isn't a developer function! That's why it wasn't inside #ifdef DEBUG!

Reply:
Oops... fixing! Thanks for reporting!